packages/binding-mqtt/src/mqtt-broker-server.ts

```diff
@@ -44,14 +46,17 @@ export default class MqttBrokerServer implements ProtocolServer {
 
     private broker: mqtt.MqttClient;
     private rejectUnauthorized: boolean;
+    private selfHost: boolean;
+    private hostedBroker: net.Server;
 
     constructor(
         uri: string,
         user?: string,
         psw?: string,
         clientId?: string,
         protocolVersion?: number,
-        rejectUnauthorized?: boolean
+        rejectUnauthorized?: boolean,
+        selfHost?: boolean
```

Comment:
Can we factor out all these parameters in a configuration interface?

Reply:
Yea, I was thinking about that and was going to do it because that constructor is barely readable now